samples/java/hibernate/src/main/java/com/google/cloud/postgres/CurrentLocalDateTimeGenerator.java

```diff
@@ -0,0 +1,13 @@
+package com.google.cloud.postgres;
```

Comment:
nit: add copyright headers to all of these classes

Reply:
Done